jetbrains-core/src-202/AwsLocalTerminalRunner.kt

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.terminal
+
+import com.intellij.openapi.project.Project
+import org.jetbrains.plugins.terminal.LocalTerminalDirectRunner
+import software.amazon.awssdk.auth.credentials.AwsCredentials
+import software.aws.toolkits.core.region.AwsRegion
+import software.aws.toolkits.core.region.toEnvironmentVariables
+import software.aws.toolkits.core.utils.getLogger
+import software.aws.toolkits.jetbrains.core.credentials.toEnvironmentVariables
+
+@Suppress("UnstableApiUsage") // TODO Remove after 2020.3 FIX_WHEN_MIN_IS_203
+class AwsLocalTerminalRunner(project: Project, private val region: AwsRegion, private val credentials: AwsCredentials) : LocalTerminalDirectRunner(project) {
```

Comment:
Is this the same as 201? Should it go in a 201-202 source set?

Reply:
it isn't 201 returns an `Array` and the method is called `getCommand`, 202 returns a `List` and the method is called `getCommands`

Reply:
ah thx, missed that

Reply:
trust me - so did I the first time ;)